### PR TITLE
Updated the README.md

### LINKED ISSUE #6: Add readme for project
Add project setup instructions in the readme

### PR DESCRIPTION
**Pull request by ShaileshKumar007**
This PR fixes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
-# cc-backend
\ No newline at end of file
+# CommClassroom-Backend
+
+A backend project for the CommClassroom Website.
+
+---
+## Requirements
+
+For development, you will only need Node.js and a node global package, Yarn, installed in your environement.
```
Comment (YashKumarVerma) [inline README.md:8]: yarn is not required for the project. works fine with npm.

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+---
+
+## Install
+
+    $ git clone https://github.com/YOUR_USERNAME/PROJECT_TITLE
+    $ cd PROJECT_TITLE
+    $ yarn install
+
+## Configure app
+
```
Comment (YashKumarVerma) [inline README.md:53]: add project specific instructions, people do not need to clone the project to run it.
use this repo's URL itself.

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+- Another setting;
+- One more setting;
+
+## Running the project
+
+    $ yarn start
+
+## Simple build for production
+
+    $ yarn build
```
Comment (YashKumarVerma) [inline README.md:66]: use NPM specific commands.
yarn can be added, but don't leave npm

File: README.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+
+### Node
+- #### Node installation on Windows
+
+  Just go on [official Node.js website](https://nodejs.org/) and download the installer.
+Also, be sure to have `git` available in your PATH, `npm` might need it (You can find git [here](https://git-scm.com/)).
+
+- #### Node installation on Ubuntu
+
+  You can install nodejs and npm easily with apt install, just run the following commands.
```
Comment (YashKumarVerma) [inline README.md:18]: not needed, if you want, can add a link to a blog that installs nodejs

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,26 @@
...
+
+    npm install
+
+Run unit tests
+
+    npm test
+
+Build everything (outputs will live in `build/`-directory)
+
+    npm run build
```
Comment (YashKumarVerma) [inline README.md:22]: this is used in typescript projects or projects with build systems. We do not need this line.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (ShaileshKumar007) [reply README.md:22]: Actually, I'm new to this, so can you guide me little bit like what should I do.

Comment (divyanshu-parihar) [reply README.md:22]: Actually what he is saying is , in typescript we compile TS to js for production , but this project is already in js so there is not need to of any type of build command atleast at this point unless team decide to use webpack or something . So you can just remove that line.

Comment (divyanshu-parihar) [reply README.md:22]: > Actually what he is saying is , in typescript we compile TS to js for production , but this project is already in js so there is not need to of any type of build command atleast at this point unless team decide to use webpack or something . So you can just remove that line.

Actually what he is saying is , in typescript we compile TS to js for production , but this project is already in js so there is not need to of any type of build command atleast at this point unless team decide to use webpack or something . So you can just remove that line.

Comment (ShaileshKumar007) [reply README.md:22]: oh ok, got it now. Thank you @divyanshu-parihar for helping me

Comment (ShaileshKumar007) [reply README.md]: > this is used in typescript projects or projects with build systems. We do not need this line.

Build command deleted.

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**